monkey/tests/unit_tests/monkey_island/cc/services/agent_plugin_service/test_agent_plugin_service.py

```diff
@@ -0,0 +1,82 @@
+import pytest
```

Comment:
These unit tests are likely unnecessary. The cyclomatic complexity of what it's testing is 1. I'll leave it up to you whether or not this code is worth maintaining, but my vote would be that it's not. There can't be a logical error, and ETE tests would catch any syntax errors.